test/integration/owner_test.rb

```diff
@@ -77,10 +77,7 @@ class OwnerTest < SystemTest
       click_button "Remove"
     end
 
-    owners_table = page.find(:css, ".owners__table")
-    within_element owners_table do
-      refute_selector(:css, "a[href='#{profile_path(@other_user)}']")
-    end
+    refute page.has_selector? "a[href='#{profile_path(@other_user)}']"
```

Comment:
:thinking: To keep it 100% the same, I think it should be 

```ruby
    refute page.has_selector? ".owners__table a[href='#{profile_path(@other_user)}']"
```

Anyway I'm ok with this as well, since it doesn't rely on surrounding CSS classes.